pola/templates/base.html

```diff
@@ -33,7 +33,7 @@
             <!-- Brand and toggle get grouped for better mobile display -->
             <div class="navbar-header">
                 <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-navbar-collapse-1" aria-expanded="false">
-                    <span class="sr-only">{% trans "Toggle navigation" %}</span>
+                    <span class="sr-only">{% trans "Nawigacja" %}</span>
```

Comment:
Odtwórz nawigacje. To pojawia się, gdy osoba niepełnosprawna dojdzie do hamburgowego menu. Ten tekst nigdy nie jest wyświetlany, a jest widoczny tylko przez czytnik dla niepełnosprawnych